src/types/types.ts

```diff
@@ -3,4 +3,31 @@ export type DefaultLoadingStatus = 'not loaded' | 'loading' | 'loaded' | 'error'
 export type DefaultLoadingState = {
   loading: DefaultLoadingStatus;
   error?: string;
-}
\ No newline at end of file
+}
+
+export interface Book {
+  id: string;
+  bookTitle: string;
+  bookAuthor: string;
+  bookDescription: string;
+  bookPrice: number;
+  bookImgUrl: string;
+}
+
```

Comment:
This file is only for global types.

Reply:
Okay I thought of keeping all the types in one place ... I have faced an issue in my project where we wanted to decouple typescript and javascript React codebase that's why I did it in this fashion